### PR TITLE
Feature/tinymce5update 3 releases 5.9 to 5.8

### PR DESCRIPTION
**Pull request by lorenzo-pomili**
Related Ticket: 

Description of Changes, it covers:

- Release TinyMCE 5.9
- Release TinyMCE 5.8.2
- Release TinyMCE 5.8.1
- Release TinyMCE 5.8

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/pages/release-notes/release-notes59.adoc [hunk 1]
```diff
@@ -362,28 +372,31 @@ For information on:
...
 
 The following features have been deprecated with the release of {productname} 5.9:
 
-* <<thespellchecker_select_languagesoption,The `spellchecker_select_languages` option>>.
-* <<thebbcodebbcodeplugin,The BBCode (`bbcode`) plugin>>
-* <<thefullpagefullpageplugin,The Full Page (`fullpage`) plugin>>
-* <<thelegacyoutputlegacyoutputplugin,The Legacy Output (`legacyoutput`) plugin>>
-* <<reminderthefreetinymcespellcheckerplugin,Reminder: The free TinyMCE Spell Checker plugin>>
+* xref:thespellchecker_select_languagesoption[The `spellchecker_select_languages` option]
+* xref:thebbcodebbcodeplugin[The BBCode (`bbcode`) plugin]
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes59.adoc:389]: This xref doesn't resolve as the heading below is missing the `thebbcodebbcodeplugin` id. e.g:
```
[[thebbcodebbcodeplugin]]
=== The BBCode (`bbcode`) plugin
```

File: modules/ROOT/pages/release-notes/release-notes59.adoc [hunk 2]
```diff
@@ -362,28 +372,31 @@ For information on:
...
+* xref:thefullpagefullpageplugin[The Full Page (`fullpage`) plugin]
+* xref:thelegacyoutputlegacyoutputplugin[The Legacy Output (`legacyoutput`) plugin]
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes59.adoc:391]: This xref doesn't resolve as the heading below is missing the `thelegacyoutputlegacyoutputplugin` id. e.g:
```
[[thelegacyoutputlegacyoutputplugin]]
=== The Legacy Output (`legacyoutput`) plugin
```

File: modules/ROOT/partials/misc/releasenotes_for_stable.adoc [hunk 3]
```diff
@@ -1 +1 @@
-NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: [TinyMCE Changelog](https://www.tiny.cloud/docs/changelog/).
+NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: link:https://www.tiny.cloud/docs/changelog/[TinyMCE Changelog].
```
Comment (lnewson) [inline modules/ROOT/partials/misc/releasenotes_for_stable.adoc:1]:
```suggestion
NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: xref:changelog.adoc[TinyMCE Changelog].
```
This should be an xref, not a link.

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/pages/release-notes/release-notes58.adoc [hunk 4]
```diff
@@ -7,18 +7,19 @@
...
-* <<upgradingtothelatestversionoftinymce5,Upgrading to the latest version of TinyMCE 5>>
+* xref:newfeatures[New features]
+* xref:enhancements[Enhancements]
+* xref:functionalitychanges[Functionality changes]
+* xref:accompanyingpremiumpluginchanges[Accompanying Premium Plugin changes]
+* xref:accompanyingpremiumskinsandiconpackschanges[Accompanying Premium Skins and Icon Packs changes]
+* xref:generalbugfixes[General bug fixes]
+* xref:deprecatedfeatures[Deprecated features]
+* xref:knownissues[Known issues]
+* xref:upgradingtothelatestversionoftinymce5[Upgrading to the latest version of TinyMCE 5]
```
[the remark this answers is not in the document]
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes58.adoc:18]: yes, fixed [here](https://github.com/tinymce/tinymce-docs/pull/2315/files#diff-7c49e18376006655b42d436d47ca3246fe30dc39246f72fbb32b064fecba7a6bR1), I'll make stack PR next time sorry

File: modules/ROOT/pages/release-notes/release-notes59.adoc [hunk 5]
```diff
@@ -272,23 +280,24 @@ The Java server-side components have been updated to the following versions:
 * `ephox-hyperlinking.war`: 2.105.4
 * `ephox-image-proxy.war`: 2.105.3
 
-These versions require Java 8 or higher. For information on the removal of Java 7 support, see: link:{baseurl}/release-notes/release-notes53/#removalofjava7support[Removal of Java 7 support for TinyMCE 5.3 and later].
+These versions require Java 8 or higher. For information on the removal of Java 7 support, see: xref:release-notes/release-notes53.adoc#removalofjava7support[Removal of Java 7 support for TinyMCE 5.3 and later].
```
[the remark this answers is not in the document]
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes59.adoc:283]: solved [here](https://github.com/tinymce/tinymce-docs/pull/2315/files#diff-bd28b4ac872746fc88f0835c1bb00679aa0a04f7a31f09f5c0017bc6e733626dR162)

File: modules/ROOT/pages/release-notes/release-notes59.adoc [hunk 6]
```diff
@@ -48,48 +49,49 @@ The following commands have been added to the table plugin to improve the usabil
...
 
-`tablecellborderstyle`:: Allows the user to set the border style on selected table cells. The available styles are set with the link:{baseurl}/plugins/opensource/table/#table_border_styles[`table_border_styles` option].
+`tablecellborderstyle`:: Allows the user to set the border style on selected table cells. The available styles are set with the xref:plugins/opensource/table.adoc#table_border_styles[`table_border_styles` option].
 
 `tablecaption`:: Toggles the caption on the selected table.
 
-`tablecellbackgroundcolor`:: Allows the user to change the background color of selected cells. The available colors are set by the link:{baseurl}/plugins/opensource/table/#table_background_color_map[`table_background_color_map` option] or link:{baseurl}/configure/content-appearance/#color_map[`color_map` option].
+`tablecellbackgroundcolor`:: Allows the user to change the background color of selected cells. The available colors are set by the xref:plugins/opensource/table.adoc#table_background_color_map[`table_background_color_map` option] or xref:configure/content-appearance.adoc#color_map[`color_map` option].
```
[the remark this answers is not in the document]
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes59.adoc:72]: done

File: modules/ROOT/pages/release-notes/release-notes59.adoc [hunk 7]
```diff
@@ -246,16 +253,17 @@ The {productname} 5.9 release includes an accompanying release of the *Premium S
 
 The following icons have been added to all premium icon packs:
 
-* New icons for new table toolbar buttons described under <<newtabletoolbarbuttonsandmenuitems,New features - New table toolbar buttons and menu items>>.
-* An icon for the <<advancedtables110,new Advanced Tables row numbering feature>>.
-* A new `language` icon for the <<newlanguagetoolbarbuttonandmenuitemincore,New language toolbar button and menu item in core>>.
+* New icons for new table toolbar buttons described under xref:newtabletoolbarbuttonsandmenuitems[New features - New table toolbar buttons and menu items].
+* An icon for the xref:advancedtables110[new Advanced Tables row numbering feature].
+* A new `language` icon for the xref:newlanguagetoolbarbuttonandmenuitemincore[New language toolbar button and menu item in core].
```
[the remark this answers is not in the document]
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes59.adoc:259]: done

File: modules/ROOT/pages/release-notes/release-notes59.adoc (hunk 1 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes59.adoc:390]: done

File: modules/ROOT/pages/release-notes/release-notes59.adoc (hunk 2 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes59.adoc:392]: done

File: modules/ROOT/partials/misc/releasenotes_for_stable.adoc (hunk 3 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/misc/releasenotes_for_stable.adoc]: done

**Review by EkimChau — APPROVED** [review]

**Pull request merged**